blog/composer.json

```diff
@@ -76,22 +77,22 @@
         "yiisoft/user": "^2.0",
         "yiisoft/validator": "^1.0",
         "yiisoft/var-dumper": "^1.0",
-        "yiisoft/view": "^8.0",
+        "yiisoft/view": "^9.0",
         "yiisoft/widget": "^2.1",
         "yiisoft/yii-bootstrap5": "^3.0@dev",
         "yiisoft/yii-console": "^2.0",
         "yiisoft/yii-cycle": "^1.0",
-        "yiisoft/yii-dataview": "^3.0@dev",
-        "yiisoft/yii-debug": "^3.0@dev",
-        "yiisoft/yii-debug-api": "^3.0@dev",
+        "yiisoft/yii-dataview": "^3.0",
+        "yiisoft/yii-debug": "*",
+        "yiisoft/yii-debug-api": "*",
         "yiisoft/yii-event": "^2.0",
         "yiisoft/yii-http": "^1.0",
         "yiisoft/yii-middleware": "^1.0",
         "yiisoft/yii-runner-console": "^2.0",
         "yiisoft/yii-runner-http": "^2.0",
         "yiisoft/yii-sentry": "dev-master",
-        "yiisoft/yii-swagger": "^2.0",
-        "yiisoft/yii-view": "^6.0"
+        "rossaddison/yii-swagger": "dev-master",               
+        "rossaddison/yii-view": "*"
```

Comment:
Why?

Reply:
This was a composer setup issue. Ignore.